libminifi/test/BufferReader.h

```diff
@@ -44,7 +44,7 @@ class BufferReader : public org::apache::nifi::minifi::InputStreamCallback {
   }
 
   int64_t process(const std::shared_ptr<org::apache::nifi::minifi::io::BaseStream>& stream) {
-    return write(*stream.get(), stream->size());
+    return static_cast<int64_t>(write(*stream.get(), stream->size()));
```

Comment:
why not `gsl::narrow`?

Reply:
Because I wanted `static_cast<size_t>(-1)` to become `-1`. I will change it to explicitly check for error.

Reply:
changed in 47358cb